dev-games/t4k-common/files/t4k-common-0.1.1-bool.patch

```diff
@@ -0,0 +1,33 @@
+Double-wrap bool in case it's a keyword, not macro, for C23.
```

Comment:
We should upstream fixes wherever possible, but this is _especially_ important if it affects the interface of something (like installed files). Please send a PR there.